package.json

```diff
@@ -58,6 +59,23 @@
     "test": "node ./node_modules/vscode/bin/test"
   },
   "contributes": {
+    "viewsContainers": {
+      "activitybar": [
+        {
+          "id": "PowerShellCommandExplorer",
+          "title": "PowerShell Command Explorer",
+          "icon": "media/PwSh.svg"
```

Comment:
if it's not too much trouble can we rename this to `pwsh.svg`?

Reply:
Done 👍